source/commands/env/export/components/hooks/PermitSDK.ts

```diff
@@ -0,0 +1,13 @@
+import { Permit } from 'permitio';
+import React from 'react';
+
+export const PermitSDK = (token: string) => {
+	return React.useMemo(
+		() =>
+			new Permit({
+				token,
+				pdp: 'http://localhost:7766',
```

Comment:
This variable should be configurable from the function arguments to allow configurable pdp URL for further usages